employee/Makefile

```diff
@@ -0,0 +1,25 @@
+CC=$(HOME)/cheri/output/sdk/bin/clang
+CFORMAT=$(HOME)/cheri/output/sdk/bin/clang-format
+CFLAGS=-fuse-ld=lld --config cheribsd-riscv64-purecap.cfg
+SSHPORT=10021
+export
+
+cfiles := $(wildcard *.c)
+examples := $(patsubst %.c,bin/%,$(cfiles))
+
+all: full_privileges read_only
+
+full_privileges: full_privileges.c employee.c
```

Comment:
Does this automatically link the two `.o` files that result from compiling the `.c` files?

Reply:
Yep

Reply:
I did not know that!